lib/rubocop/cop/performance/ancestors_include.rb

```diff
@@ -35,7 +35,9 @@ def on_send(node)
         def autocorrect(node)
           ancestors_include_candidate?(node) do |subclass, superclass|
             lambda do |corrector|
-              corrector.replace(node, "#{subclass.source} <= #{superclass.source}")
+              subclass_source = subclass ? subclass.source : 'self.class'
```

Comment:
The receiver of `ancestors` is already a class.

```suggestion
              subclass_source = subclass ? subclass.source : 'self'
```

Reply:
Ah...sure 💦  I updated it. Thank you for your review!